src/presenter/board-presenter.js

```diff
@@ -0,0 +1,105 @@
+import SiteFilters from '../view/site-menu-filters.js';
+import MainSort from '../view/main-sort.js';
+import TripList from '../view/trip-events-list.js';
+import PageHeader from '../view/page-header.js';
+import MainNavigation from '../view/menu-navigation.js';
+import TripCost from '../view/trip-info-cost.js';
+import InfoMain from '../view/trip-info-main.js';
+import PageMain from '../view/page-main.js';
+import NoData from '../view/no-data.js';
+import TripPresenter from './trip-presenter.js';
+import { render, updateItem } from '../render.js';
+
+export default class BoardPresenter {
+  #tripContainer = null;
+
+  #HeaderComponent = new PageHeader();
+  #MainComponent = new PageMain();
+  #TripList =  new TripList();
+  #sortComponent = new MainSort();
+  #noTripsComponent = new NoData();
+
+  #trips = [];
+  #tripPresenter = new Map();
```

Comment:
```suggestion
  #tripPresenters = new Map();
```
создаешь дополнительную нагрузку на систему распознавания проверяющего/сопровождающего сотрудника